WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40383,553 +40411,6 @@ e_latin_empire = {
-	
-	capital = 774 # Jerusalem
-	
-	title = "HIGH_PATRIARCH"
-	foa = "ECUMENICAL_PATRIARCH_FOA"
-	short_name = yes
-	
-	allow = {
-		always = no # Only by special decision
-	}
-
-	# Always exists
-	landless = yes
-	
-	# Controls a religion
-	controls_religion = binitarian
-	
-	religion = binitarian 
-	
-	# Cannot be held as a secondary title
-	primary = yes
-	
-	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
-	
-	# Regnal names
-	male_names = {
-		Alexios Alexandros Anastasios Anthimos Athanasios Antonios Basileios Gabriel Dionysios Dositheos 
-		Eustathios Eustratios Euthymios Gabriel Gennadios Georgios Gerasimos Germanos Gregorios Ieremias 
-		Ignatios Ioakeim Ioannes Ioseph Isidoros Kallinikos Kallistos Konstantinos Kosmas Kyrillos Leon 
-		Leontios Loukas Makarios Manuel Markos Matthaios Maximos Meletios Methodios Metrophanes Michael 
-		Neophytos Nephon Nikephoros Niketas Nikolaos Pavlos Petros Photios Polykarpos Sergios Stephanos 
-		Sophronios Theodoros Theodosios Theodotos Theophylaktos Thomas Timotheos
-	}
-}
-
-d_ebionites = {
-	color={ 153 153 153 }
-	color2={ 220 220 0 }
-	
-	capital = 774 # Jerusalem
-	
-	title = "EBIONITES_RABI"
-	foa = "HEAD_KOHEN_GADOL_FOA"
-	short_name = yes
-	
-	allow = {
-		always = no # Only by special decision
-	}
-
-	# Always exists
-	landless = yes
-	
-	# Controls a religion
-	controls_religion = ebionites
-	
-	religion = ebionites
-	
-	# Cannot be held as a secondary title
-	primary = yes
-	
-	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
-	
-}
-
-d_valentinianism = {
-	color={ 153 153 153 }
-	color2={ 220 220 0 }
-	
-	capital = 333 # Rome
-	
-	title = "ADAM"
-	foa = "POPE_FOA"
-	short_name = yes
-	
-	allow = {
-		always = no # Only by special decision
-	}
-
-	# Always exists
-	landless = yes
-	
-	# Controls a religion
-	controls_religion = valentinianism
-	
-	religion = valentinianism
-	
-	# Cannot be held as a secondary title
-	primary = yes
-	
-	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
-	
-}
-
-d_priscillianism = {
-	color={ 153 153 153 }
-	color2={ 220 220 0 }
-	
-	capital = 198 # Valladolid
-	
-	title = "POPE"
-	foa = "POPE_FOA"
-	
-	short_name = yes
-	
-	allow = {
-		always = no # Only by special decision
-	}
-
-	# Always exists
-	landless = yes
-	
-	# Controls a religion
-	controls_religion = priscillianism
-	
-	religion = priscillianism
-	
-	# Cannot be held as a secondary title
-	primary = yes
-	
-	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
-}
-
-d_sethian = {
- 	color={ 0.0 0.0 0.2 }
- 	color2={ 255 0 128 }
- 	
- 	capital = 802 # Alexandria
- 
- 	title = "HEAD_SNAKE_KEEPER"
- 	foa = "POPE_FOA"
- 	
- 	short_name = yes
-	
-	allow = {
-		always = no # Only by special decision
-	}
- 
- 	# Always exists
- 	landless = yes
- 	
- 	# Controls a religion
- 	controls_religion = sethian
- 
- 	religion = sethian
-
- 	# Cannot be held as a secondary title
-	primary = yes
- 	
- 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
-  }
```

Comment:
I'm not going to this hassle for you @TurtleShroom, especially as I can't modify directly while you can. The browser lags and slugs on the page. You can see the changes in this pull request, revert that. Good that you realise how much work this is which is wasted on moving these files though in these pull-requests.